src/components/BookItem/ItemStart/ItemStart.css

```diff
@@ -0,0 +1,54 @@
+.blue-text {
+  color: #4386bf;
+}
+
+.item-start {
+  display: flex;
+  flex-direction: column;
+  align-self: stretch;
+}
+
+.item-main {
+  /* margin-bottom: 20px; */
```

Comment:
- [ ] I think you have forgotten to remove this commented line, Kindly get rid of it, It is always a good practice to keep your code clean from such comments.